extensions/http/jetty/src/main/java/org/eclipse/dataspaceconnector/extension/jetty/JettyService.java

```diff
@@ -178,14 +179,16 @@ private ServerConnector httpsServerConnector(int port) {
         var httpConnectionFactory = new HttpConnectionFactory(httpsConfiguration);
         var sslConnectionFactory = new SslConnectionFactory(contextFactory, HttpVersion.HTTP_1_1.asString());
         var sslConnector = new ServerConnector(server, sslConnectionFactory, httpConnectionFactory);
+        sslConnector.setName(name);
```

Comment:
the `name` property is already set on the connector (see `JettyService` at line 92), so no need to set it here neither in the `httpServerConnector` method

Reply:
Interesting - I thought, that it was empty as a debugged into it. Will take a look again

Reply:
The JettyMicrometerConfiguration was executed first, before the name and port were configured. So I reworked it as discussed offline.